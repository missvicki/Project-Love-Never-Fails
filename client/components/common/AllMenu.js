import React from "react";
import { Menu } from "semantic-ui-react";
import { NavLink } from "react-router-dom";

const AllMenu = props => {
  const { items } = props;
  return (
    <Menu.Menu position="right">
      <Menu.Item name="menu" className={items}>
        <NavLink to="/" className={items}>
          OUR STORY
        </NavLink>
      </Menu.Item>
      <Menu.Item name="mission" className={items}>
        <NavLink to="/" className={items}>
          MISSION
        </NavLink>
      </Menu.Item>
      <Menu.Item name="children" className={items}>
        <NavLink to="/" className={items}>
          OUR CHILDREN
        </NavLink>
      </Menu.Item>
      <Menu.Item name="donate" className={items}>
        <NavLink to="/" className={items}>
          DONATE
        </NavLink>
      </Menu.Item>
    </Menu.Menu>
  );
};

export default AllMenu;
