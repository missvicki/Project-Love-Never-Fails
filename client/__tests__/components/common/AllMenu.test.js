import React from "react";
import { shallow } from "enzyme";
import AllMenu from "../../../components/common/AllMenu";

describe("<AllMenu />", () => {
  const props = {
    items: "item"
  };
  const wrapper = shallow(<AllMenu {...props} />);
  it("should render AllMenu component without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
