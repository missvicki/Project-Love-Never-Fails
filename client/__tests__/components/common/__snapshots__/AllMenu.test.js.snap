// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AllMenu /> should render AllMenu component without crashing 1`] = `
<MenuMenu
  position="right"
>
  <MenuItem
    className="item"
    name="menu"
  >
    <NavLink
      className="item"
      to="/"
    >
      OUR STORY
    </NavLink>
  </MenuItem>
  <MenuItem
    className="item"
    name="mission"
  >
    <NavLink
      className="item"
      to="/"
    >
      MISSION
    </NavLink>
  </MenuItem>
  <MenuItem
    className="item"
    name="children"
  >
    <NavLink
      className="item"
      to="/"
    >
      OUR CHILDREN
    </NavLink>
  </MenuItem>
  <MenuItem
    className="item"
    name="donate"
  >
    <NavLink
      className="item"
      to="/"
    >
      DONATE
    </NavLink>
  </MenuItem>
</MenuMenu>
`;
